src/v1.js

```diff
@@ -136,19 +138,27 @@ api.declare({
   ].join('\n'),
 }, async function(req, res) {
   let prefix = req.query.prefix;
+  let continuationToken  = req.query.continuationToken || undefined;
+  let limit         = parseInt(req.query.limit || 10, 10);
+  let Client = this.Client;
+  let resolver = this.resolver;
 
-  // Load all clients
-  // TODO: as we acquire more clients, perform the prefix filtering in Azure
-  let clients = [];
-  await this.Client.scan({}, {
-    handler: client => {
+  let response = {clients: []};
+
+  while (true) {
```

Comment:
The `while(true)` thing here isn't necesary... I think it's okay to return `{clients: [], continuationToken: '...'}`.

Mostly, I think it's better than risk the a timeout, which is what `continuationToken` is intended to prevent.

Reply:
Hm, I thought the opposite, but you make a good point about timeouts -- if we were to have a filter that matched only one client, and a LOT of clients, then it might take more than 30s to find that one client.  We have a limit of 30s for each HTTP transaction.

@tuhina2020 rather than making you reverse that change, I'll push a commit that does it.  Sorry!